pymodbus/server/reactive/main.py

```diff
@@ -368,6 +369,9 @@ def factory(  # pylint: disable=dangerous-default-value,too-many-arguments
         :              refer corresponding servers documentation
         :return: ReactiveServer object
         """
+        if isinstance(server, Enum):
```

Comment:
I am still trying to learn about the python ecosystem. I don't see a test for this method and I am not sure if we have a test for this reactive server. Do we have one?

Reply:
If you see the other PR open, there is a pending task to add tests. This PR will be closed in favor of that